lib/build.js

```diff
@@ -455,19 +455,21 @@ const parseAct = config => (act_str, silent = false) => {
                  * preceded by:
                  *  - ', ' or ','
                  *  - '('
+                 *  - '['
                  *  - the start of a line
                  *  - a tab
                  *  - a space
                  * and followed by:
                  *  - ','
                  *  - ')'
+                 *  - ']'
                  *  - the end of a line
                  *  - a tab
                  *  - a space
                  *
-                 *  example here: https://regex101.com/r/ZubkN1/3
+                 *  example here: https://regex101.com/r/ZubkN1/4
                  */
-                "from": new RegExp(`(, ?|\\(|^|\\t| )(${l.split(":=")[0].trim()})(,|\\)|$|\\t| )`, "gm"),
+                "from": new RegExp(`(, ?|\\(|\\[|^|\\t| )(${l.split(":=")[0].trim()})(,|\\)|$|\\t|\\]| )`, "gm"),
```

Comment:
Why do you need to doube escape with `\\[`? The regex in the link doesn't need to do that for some reason.

Reply:
In the link, a regex is constructed with `/<regex>/`
here an object of "class" RegExp is constructed and a String passed to its constructor
in a String the `\` character is an escape character used for place strange characters in a string: e.g. `\t` is a tab `\n` new line, etc. so in order to type the '\' character in a string as exactly that character you need to escape it: `\\`
the final string object will only contain `\[` in this case